
This patch:

1. Fixes missing SCMI clock for cpu little cores
2. Fixes assigned clock rates: SCMI CPU clocks should default to 816 MHz, which is what the firmware defaults to.


diff --speed-large-files --no-dereference --minimal -Naur linux-6.3/arch/arm64/boot/dts/rockchip/rk3588s.dtsi linux-6.3/arch/arm64/boot/dts/rockchip/rk3588s.dtsi
--- linux-6.3/arch/arm64/boot/dts/rockchip/rk3588s.dtsi	2023-04-26 20:25:54.722066092 +0200
+++ linux-6.3/arch/arm64/boot/dts/rockchip/rk3588s.dtsi	2023-04-26 20:24:36.235402322 +0200
@@ -534,10 +534,12 @@
 
 			scmi_clk: protocol@14 {
 				reg = <0x14>;
-				assigned-clocks = <&scmi_clk SCMI_CLK_CPUB01>,
+				assigned-clocks = <&scmi_clk SCMI_CLK_CPUL>,
+						  <&scmi_clk SCMI_CLK_CPUB01>,
 						  <&scmi_clk SCMI_CLK_CPUB23>;
-				assigned-clock-rates = <1200000000>,
-						       <1200000000>;
+				assigned-clock-rates = <816000000>,
+						       <816000000>,
+						       <816000000>;
 				#clock-cells = <1>;
 			};
 
